client/src/gamedata/authors.json

```diff
@@ -147,6 +147,11 @@
       "websites": [
         "https://www.linkedin.com/in/kstasi/"
       ]
+    },
+    "AshiqAmien": {
+      "name": "Ashiq Amien",
```

Comment:
Are all these meant to be arrays instead of single key values ?

Reply:
Oh thanks for the catch, changing now